cli/src/clients/metas_client.rs

```diff
@@ -116,14 +121,22 @@ impl MetasClient {
         Ok(Self {
             inner: raw_client,
             base_url: env.meta_base_url.clone(),
+            bearer_token: env.bearer_token.clone(),
+            request_timeout: env.request_timeout.clone(),
         })
     }
 
     /// Prepare a request builder for the given method and path.
     fn prepare(&self, method: reqwest::Method, path: Url) -> reqwest::RequestBuilder {
-        // TODO: Inject the secret token when available.
-        //.bearer_auth(&self.secret_token);
-        self.inner.request(method, path)
+        let request_builder = self
+            .inner
+            .request(method, path)
+            .timeout(self.request_timeout);
```

Comment:
Do you think we should also rely on `connect_timeout` here?

Reply:
I am less sure about doing it here, but my immediate reaction is sure – it should catch most misconfigurations. We can always add a service/response timeout later. That may make sense to be quite tight for the meta service but you never know, over the internet.

Reply:
I came up with a solution to split the defaults – there is now a `connect_timeout` with a universal default of 5s, plus a separate `request_timeout` with a silent default of 10s that is only applied to the `MetasClient`. Seems like a good compromise to me but you can well argue it's adding a lot of noise.